### ISSUE TITLE
Circular import in external_tools

### ISSUE THREAD

**Issue by Kamuish** [iastro-pt/sBART#23]
When trying to 
`from SBART.utils.paths_tools.Load_RVoutputs import find_RVoutputs`

we get:

```
  File "/data/work/amiguel/env/lib/python3.8/site-packages/SBART/utils/paths_tools/Load_RVoutputs.py", line 3, in <module>
    from SBART.data_objects.RV_outputs import RV_holder
  File "/data/work/amiguel/env/lib/python3.8/site-packages/SBART/data_objects/__init__.py", line 7, in <module>
    from .DataClass import DataClass
  File "/data/work/amiguel/env/lib/python3.8/site-packages/SBART/data_objects/DataClass.py", line 18, in <module>
    from SBART.utils.paths_tools.Load_RVoutputs import find_RVoutputs
ImportError: cannot import name 'find_RVoutputs' from partially initialized module 'SBART.utils.paths_tools.Load_RVoutputs' (most likely due to a circular import) (/data/work/amiguel/env/lib/python3.8/site-packages/SBART/utils/paths_tools/Load_RVoutputs.py)
```

## Temporary solution:
import the Dataclass right before this:
```
from SBART.data_objects import DataClass
from SBART.utils.paths_tools.Load_RVoutputs import find_RVoutputs
```

**Comment by Kamuish** [comment]
The find_RVoutputs function is simply a wrapper for the RV_holder.load_from_disk

Actions:

- Change all occurrences of this usage
- Move the find_RVoutputs to the external_tools interface

**Comment by Kamuish** [comment]
Fixed in v0.1.6 (at issue-closing time only up on `dev` branch
